### PR TITLE
Update for better web teleop ui

### PR DESCRIPTION
**Pull request by RoboEats**
screenshot from 2019-01-29 13 28 12
screenshot from 2019-01-29 13 28 23

### CODE REVIEW

**Comment by Kietyo** [comment]
lgtm

**Review by Kietyo — APPROVED** [review]
lgtm

File: web_teleop/package.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
   <!-- One maintainer tag required, multiple allowed, one person per tag -->
   <!-- Example:  -->
   <!-- <maintainer email="jane.doe@example.com">Jane Doe</maintainer> -->
-  <maintainer email="team2@todo.todo">team2</maintainer>
+  <maintainer email="kiet@todo.todo">kiet</maintainer>
```
Comment (SummerNinja) [inline web_teleop/package.xml]: We should leave this line as team2@todo.todo for the version that's merged

File: web_teleop/nodes/actuators_node.py [hunk 2]
```diff
@@ -21,45 +18,50 @@ def __init__(self):
...
     def handle_set_torso(self, request):
+        # type: (SetTorsoRequest) -> None
         # TODO: move the torso to the requested height
+        print "web_teleop/set_torso: setting height to %.2f" % request.height
         self._torso.set_height(request.height)
         return SetTorsoResponse()
-    
+
     def handle_set_head(self, request):
+        # type: (SetHeadRequest) -> None
```
Comment (SummerNinja) [inline web_teleop/nodes/actuators_node.py:33]: Same as above

File: web_teleop/nodes/actuators_node.py [hunk 3]
```diff
@@ -21,45 +18,50 @@ def __init__(self):
         self._head = robot_api.Head()
         self._gripper = robot_api.Gripper()
         self._arm = robot_api.Arm()
+
     def handle_set_torso(self, request):
+        # type: (SetTorsoRequest) -> None
```
Comment (SummerNinja) [inline web_teleop/nodes/actuators_node.py:26]: Remove commented out code (this is also a really small thing so I can live with this)

File: web_teleop/frontend/src/web-teleop-app.html [hunk 4]
```diff
@@ -16,268 +19,238 @@
...
+
     /**
      * @customElement
      * @polymer
      */
     class WebTeleopApp extends Polymer.Element {
+      ready() {
+        super.ready();
+        console.log("hostname: " + window.location.hostname);
+        // this.hostname = window.location.hostname;
```
Comment (SummerNinja) [inline web_teleop/frontend/src/web-teleop-app.html]: Remove commented out line (I can also technically live with this though)

**Review by SummerNinja — CHANGES_REQUESTED** [review]
LGTM, the one thing was leaving the maintainer the same in the package.xml and a few optional changes, but otherwise it looks good

File: web_teleop/nodes/actuators_node.py (hunk 2 above)
Comment (Kietyo) [reply web_teleop/nodes/actuators_node.py:33]: this is a type hint

File: web_teleop/nodes/actuators_node.py (hunk 3 above)
Comment (Kietyo) [reply web_teleop/nodes/actuators_node.py:26]: this is a type hint

**Pull request merged**